core/station/impl/src/core/validation.rs

```diff
@@ -199,9 +199,10 @@ impl EnsureExternalCanister {
     pub fn is_external_canister(
         principal: Principal,
     ) -> Result<(), ExternalCanisterValidationError> {
+        // todo: look into Asset repository and exclude the ledger_canister_id's
```

Comment:
This should still be added no? maybe instead of having this hardcoded giving no flexibility to the admins, we should have a deny list that is mutable 🤔